packages/TelegramClientTests-UI.package/TCTUChatsListTests.class/instance/testChatOrderUpdatesIfPositionChanges.st

```diff
@@ -0,0 +1,9 @@
+testing
+testChatOrderUpdatesIfPositionChanges
+
+	| chat |
+	chat := self core chatsHandler chats last.
+	chat position: 0.
```

Comment:
maybe access the last first one and use its position as a reference

Reply:
The problem is that no chat has a position so I cant use any as a reference. Should we give every chat a position or just leave it this way?

Reply:
maybe write a comment for that